src/Components/Movies.js

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios';
+import { Component } from 'react';
+import "./NavBar.css"
+
+class Movies extends Component {
+    state = { movies: [], movieName: "", movie: {} }
+
+    getAllMovies = async () => {
+        try {
+            const res = await axios.get(`https://ghibliapi.herokuapp.com/films`)
```

Comment:
it's not best practice to use backticks when your aren't doing string interpolation